File: examples/simp/Python/predictor/predictor2.py
# Import the library
import LinkPredPython as lpp
# Create a prtedictor object
p = lpp.Predictor();
# Load network from file
p.loadnet("Zakarays_Karate_Club.edges");
esv = p.predAllKAB();
# Print the scores
for es in esv:
	print(es.i + "\t" + es.j + "\t" + "{:.4f}".format(es.score));

File: examples/simp/CPP/predictor/predictor2.cpp
#include <linkpred.hpp>
#include <iostream>

using namespace LinkPred::Simp;

int main() {
	Predictor p;
	p.loadnet("Zakarays_Karate_Club.edges");
	auto esv = p.predAllKAB();
	for (auto it = esv.begin(); it != esv.end(); ++it) {
		std::cout << it->i << "\t" << it->j << "\t" << it->score << std::endl;
	}

	return 0;
}
